docker-compose.yml

```diff
@@ -86,6 +88,13 @@ services:
     networks:
       - frontend
       - backend
+    deploy:
```

Comment:
Super helpful add!  I was focused on the LLM/Ollama performance and didn't think about the vectorization aspect!  Thanks.